initializers/ip_addresses.yml

```diff
@@ -0,0 +1,18 @@
+
+#- address: 10.1.1.1/24
+#  device: router1
+#  family: 4
+#  interface: Loopback0
+#  status: 1
+#  vrf: vrf1
+#- address: 10.1.1.2/24
+#  device: switch1
+#  family: 4
+#  interface: Vlan5
+#  status: 1
+#- address: 10.1.1.10/24
+#  description: reserved IP
+#  family: 4
+#  status: 2
```

Comment:
Would it be possible to use the actual word here, i.e. `Active`, `Reserved`, `Deprecated`, `DHCP`?

See https://github.com/netbox-community/netbox-docker/blob/master/startup_scripts/110_devices.py#L52-L55 for how this was made for the `face` property in `devices.yml`.